spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -146,6 +157,27 @@ public BaseDeleteOrphanFilesSparkAction deleteWith(Consumer<String> newDeleteFun
     return this;
   }
 
+  @Override
+  public BaseDeleteOrphanFilesSparkAction actualFilesTable(String tableName) {
+    ValidationException.check(
+        spark().catalog().tableExists(tableName),
+        "actualFilesTable `" + tableName + "` does not exist");
+
+    try {
+      StructType schema = spark().table(tableName).schema();
+      StructField filePathField = schema.apply("file_path");
+      ValidationException.check(
+          filePathField.dataType() == DataTypes.StringType,
+          "actualFilesTable `" + tableName + "` - `file_path` column is not a string type");
```

Comment:
Nit: Normally we use plain english sentences. Like <error> <cause>.

So maybe here it could be `Invalid schema for actual files table - `file_path` column is not a string type`.